ykllvmwrap/src/ykllvmwrap.cc

```diff
@@ -17,18 +18,27 @@
 #include "llvm/Support/FormattedStream.h"
 #include "llvm/Support/SourceMgr.h"
 #include "llvm/Support/TargetSelect.h"
+#include "llvm/Support/raw_ostream.h"
 #include "llvm/Transforms/IPO/PassManagerBuilder.h"
 #include "llvm/Transforms/Utils/ValueMapper.h"
 
 #include <dlfcn.h>
 #include <err.h>
+#include <filesystem>
 #include <link.h>
 #include <stdlib.h>
 #include <string.h>
+#include <unistd.h>
 
 #include "jitmodbuilder.h"
 #include "memman.h"
 
+// When we create a compilation unit for our JIT debug info, LLVM forces us to
+// choose a language from one of those "recognised" by the DWARF spec (see
+// `Dwarf.def` in the LLVM sources). If you choose a value out of range, then an
+// LLVM assertion will fail, so we have to pretend to be another language.
+#define YKJIT_DWARF_LANG dwarf::DW_LANG_Cobol74
```

Comment:
Is this comment saying "we don't just pick one at random, but we pick one that's highly unlikely to be used in any system we're part of"?

Reply:
Yeah, kind of. That and `Cobol74` amuses me.

Reply:
Is the "we shouldn't clash with something else" property important? It's probably worth clarifying in the comment either way.

[I have no attachment to `Cobol74` either way ;)]

Reply:
It doesn't really matter.

 - It's just a field that identifies what language compiler generated the code. I've never seen it displayed outside of `dwarfdump`.
 - There can be no clash. Just as a linker can link objects compiled by the same compiler, so can `ExecutionEngine` load code generated by the same "language" as the AOT code was compiled with.

Reply:
OK.